obs_text_source.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <chrono>
+#include <string>
+
+// Add `_path` as a text source in obs to get an onscreen queso-queue timer.
+class ObsTextSource {
+  public:
+    void UpdateTime(std::chrono::seconds seconds);
+
+  private:
+    void WriteMessage(std::string message);
+    std::string _currentMessage;
```

Comment:
Hm, think we really need the cache?

Reply:
I put that in before I knew that Chat::Listen() only looped about once a second. So it's not really needed, but I figured reducing disk read/writes when possible was still useful. If you like I can remove it.

Reply:
Yeah, let's get rid of it.

Can you squash your fixup commit(s) into the original one, so only one commit gets added to the history? I think I turned off squash-on-merge for this project. LMK if you want tips on doing so, not everyone is used to rebase -i.

Reply:
Done.